game/database/schema/sfx.lua

```diff
@@ -8,5 +8,6 @@ for _,file in ipairs(filelist) do
 end
 
 return {
-  { id = 'filename', name = "Filename", type = 'enum', options = files }
+  { id = 'filename', name = "Filename", type = 'enum', options = files },
+  { id = 'polyphony', name = "Polyphony", type = 'integer', range = {1} }
```

Comment:
I don't think this should be a part of the schema.

Reply:
Where would it go then?

Reply:
Where else is a proper place to put this then?

Reply:
Shouldn't this be a constant? I find it strange that we want such a fine tuned cache for each sfx.

Reply:
We could move it to a global config variable then, but it would still be in the schema of something, because data-driving it should be better than hard-coding it.

But, from a sound design perspective, you might actually want to enforce a limit to the polyphony of certain sounds if, for instance, they are too annoying or do not overlap well. To support this argument, all audio middleware systems I've studied before allow you set a per-sample polyphony, so it's probably something they consider worthwhile.

Reply:
Alright, then it's ok the way it is